module/platform/SimscapeUDP/README.md

```diff
@@ -0,0 +1,14 @@
+SimscapeUDP
+======
+
+## Description
+    Middleware for matlab and NUbots codebase
+
+## Usage
+    Add to a Role to pass ServoTargets messages to the Matlab NUbots/Simscape which then simulates the robot. The simulation also generates sensor data from the Servo and emits them as Servos messages.
+
+## Emits
+    DarwinSensors/Servos
+    ServoTargets in the UDP scope
+
+## Dependencies
```

Comment:
Should we include MATLAB as a dependency?

Also, this only worked with ubuntu 20.04 right? that should probably be on here somewhere I think

Reply:
MATLAB is not a dependency for this part of the project, but I will hyperlink to the part that is dependent on it.